### PR TITLE
Complete feature mark to the field type

### PR DESCRIPTION
**Pull request by nguyentrieutien1**
Related Tickets
[#TicketID](https://edu-redmine.sun-asterisk.vn/issues/73662)
WHAT (optional)
Create feature mark favorite to the field type
HOW
WHY (optional)
Evidence (Screenshot or Video)

### CODE REVIEW

**Comment by nguyentrieutien1** [comment]
ready

File: app/controllers/fields_controller.rb [hunk 1]
```diff
@@ -1,10 +1,21 @@
...
+    return if @field
+
+    flash[:danger] = t("field_types.not_found")
+    redirect_to root_path
   end
-  
+
   def index
     @pagy, @fields = pagy Field.all, items: Settings.PERPAGE_8
   end
```
Comment (manhhnee) [inline app/controllers/fields_controller.rb]: Sao anh nhét index vào private zậy

File: config/locales/views/vi.yml [hunk 2]
```diff
@@ -20,6 +20,9 @@ vi:
     title: "Danh sách sân bóng đá"
     empty: "Không có loại trường nào được hiển thị"
     search_placeholder: "Tìm kiếm tại đây"
+    favorite_success: "Yêu thích thành công"
+    favorite_fail: "Yêu thích thất bại"
+    search_placeholder: "Tìm kiếm tại đây"
```
Comment (manhhnee) [inline config/locales/views/vi.yml:25]: Thừa 1 cái search_placeholder nè anh

**Review by manhhnee — COMMENTED** [review]
comment

**Pull request merged**